File: misc/js/templates/cxItemMod.js
var template = template || {};

template.cxItemMod = function(name, price) {
	return	"<li class=\"mod\">\r\n"+
			"\t<span class=\"name\">"+(name)+"</span>\r\n"+
			"\t<span class=\"price\">"+(price)+"</span>\r\n"+
			"</li>";
};

File: server/Checkout Express/src/templates/CxItemMod.java
package templates;

public class CxItemMod {
	public static String run(Object name, int price) {
		return	"<li class=\"'mod\"'>"+
			"\t<span class=\"'name\"'>"+(name)+"</span>"+
			"\t<span class=\"'price\"'>"+(price)+"</span>"+
			"</li>";
	}
}